DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/EditFilters.cshtml

```diff
@@ -0,0 +1,152 @@
+                                css-class="nhsuk-u-width-one-half"
+                                default-option="All"
+                                select-list-options="@Model.CustomisationOptions" />
+
+              </div>
+
+              <div class="nhsuk-radios__item">
+                <input name="@nameof(Model.FilterType)"
+                       class="nhsuk-radios__input"
+                       id="course-filter-type-2"
+                       type="radio"
+                       value="@CourseFilterType.CourseCategory"
+                       aria-controls="conditional-course-filter-type-2"
+                       aria-expanded="@(categorySelected ? "true" : "false")"
+                       @(categorySelected ? "checked" : "")>
+                <label class="nhsuk-label nhsuk-radios__label" for="course-filter-type-2">
+                  Course category
+                </label>
+              </div>
+              <div class="nhsuk-radios__conditional @(categorySelected ? "" : "nhsuk-radios__conditional--hidden")" id="conditional-course-filter-type-2">
+                <vc:select-list asp-for="@nameof(Model.CourseCategoryId)"
+                                label="Choose course category from the list below"
+                                value="@Model.CourseCategoryId.ToString()"
+                                hint-text=""
+                                deselectable="true"
+                                css-class="nhsuk-u-width-one-half"
+                                default-option="All"
+                                select-list-options="@Model.CourseCategoryOptions" />
+              </div>
+            </div>
+          </fieldset>
+        </div>
+      } else {
+        <input type="hidden" asp-for="FilterType" value="@CourseFilterType.CourseCategory" />
+        <vc:select-list asp-for="@nameof(Model.CustomisationId)"
+                        label="Choose a course filter"
+                        value="@Model.CustomisationId.ToString()"
+                        hint-text=""
+                        deselectable="true"
+                        css-class="nhsuk-u-width-one-half"
+                        default-option="All"
+                        select-list-options="@Model.CustomisationOptions" />
+      }
+
+      <vc:date-range-input id="date-range"
+                           label="Reporting date range"
+                           start-day-id="@nameof(Model.StartDay)"
+                           start-month-id="@nameof(Model.StartMonth)"
+                           start-year-id="@nameof(Model.StartYear)"
+                           end-day-id="@nameof(Model.EndDay)"
+                           end-month-id="@nameof(Model.EndMonth)"
+                           end-year-id="@nameof(Model.EndYear)"
+                           end-date-checkbox-id="@nameof(Model.EndDate)"
+                           end-date-checkbox-label="End date"
+                           css-class="date-range-css"

```

Comment:
Is this CSS needed as a param?

Reply:
We should keep it for consistency with the other components, but it doesn't need a value, and this specific class has no styles associated with it.